docs/sharing/sharing.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 6
+slug: /sharing
+---
+
+# Sharing
+
+Sharing datasets and projects allows for a more efficient workflow because granting access to other members enables them to simultaneously **edit and/or view your datasets or projects**. 
+
+::::info
+Sharing **does not duplicate** your data, only grants access to it.
+::::
+
+## **Managing teams and members**
+
+In the [Settings](../workspace/settings.md) you can find the Teams you are part of, and you can see the members and their role. Teams are little departments of Organizations.
```

Comment:
@NNoemi1 
Can we rephrase "_Teams are little departments of Organizations_" to "Teams can represent departments within an organization and allow you to group members in a way that suits your workflow"